x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -113,8 +113,8 @@
     "advancedSettings.field.settingIsUnsaved": "设备当前未保存。",
     "advancedSettings.field.unsavedIconLabel": "未保存",
     "advancedSettings.form.cancelButtonLabel": "取消更改",
-    "advancedSettings.form.clearNoSearchResultText": " (清除搜索) ",
-    "advancedSettings.form.clearSearchResultText": " (清除搜索) ",
+    "advancedSettings.form.clearNoSearchResultText": "（清除搜索）",
```

Comment:
I'm always confused with this symbol. Which one is the good one?

Reply:
What do you mean? which symbol